test/integration/docker_build_test.exs

```diff
@@ -115,8 +115,32 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
                      docker_servername: ""
                    })
         end)
-
       assert log =~ "STEP 1/1 : FROM alpine:latest"
     end
   end
+
+  describe "container listens on the specified network ports" do
+    test "inspecting container by container id" do
+      instructions = [
+        {"FROM", "alpine:latest"},
+        {"VOLUME", @cwd <> ":/data"},
```

Comment:
can you delete `VOLUME, RUN, CMD` steps from the instructions as they are not necessary?